File: zebra/priv/legacy_repo/migrations/20251119104017_add_organization_created_index_at_jobs_table.exs
defmodule Zebra.LegacyRepo.Migrations.AddOrganizationCreatedIndexAtJobsTable do
  use Ecto.Migration
  @disable_migration_lock true
  @disable_ddl_transaction true

  def change do
    create index(:jobs, [:organization_id, :created_at],
      name: "index_jobs_on_organization_created_expires_is_null",
      concurrently: true,
      where: "expires_at IS NULL"
    )
  end
end

File: github_hooks/db/migrate/20251119104017_add_organization_created_index_at_jobs_table.rb
class AddOrganizationCreatedIndexAtJobsTable < ActiveRecord::Migration[6.1]
  disable_ddl_transaction!

  def change
    add_index :jobs,
              %i[organization_id created_at],
              name: "index_jobs_on_organization_created_expires_is_null",
              algorithm: :concurrently,
              where: "expires_at IS NULL",
              if_not_exists: true
  end
end
